docs/user_manual.md

```diff
@@ -878,6 +879,157 @@ datum stats -p test_project
+```
+
+Validation Result:
+
+<details>
+
+``` bash
+{
+    'statistics': {
+        ## common statistics
+        'label_distribution': {
+            'defined_labels': <dict>,   # <label:str>: <count:int>
+            'undefined_labels': <dict>
+            #  <label:str>: {
+            #    'count': <int>,
+            #    'items_with_undefined_label': [(<item.id>, <item.subset>), ]
+            #  }
+        },
+        'attribute_distribution': {
+            'defined_attributes': <dict>,
+            #  <label:str>: {
+            #    <attribute:str>: {
+            #      'distribution': {<attr_value:str>: <count:int>, },
+            #      'items_missing_attribute': [(<item.id>, <item.subset>), ]
+            #    }
+            #  }
+            'undefined_attributes': <dict>
+            #  <label:str>: {
+            #    <attribute:str>: {
+            #      'distribution': {<attr_value:str>: <count:int>, },
+            #      'items_with_undefined_attr': [(<item.id>, <item.subset>), ]
+            #    }
+            #  }
+        },
+        'total_ann_count': <int>,
+        'items_missing_annotation': <list>, # [(<item.id>, <item.subset>), ]
+
+        ## statistics for classification task
+        'items_with_multiple_labels': <list>, # [(<item.id>, <item.subset>), ]
+
+        ## statistics for detection task
+        'items_with_invalid_value': <dict>,
+        #  '(<item.id>, <item.subset>)' : {
+        #    <ann.id> : [ <property:str>, ]
+        #  }
+        #  properties: 'x', 'y', 'width', 'height',
+        #              'area(wxh)', 'ratio(w/h)', 'short', 'long'
+        'items_with_negative_length': <dict>,
+        #  '(<item.id>, <item.subset>)' : {
+        #    <ann.id> : { <'width'|'height'>: <value>, }
+        #  }
+        'bbox_distribution_in_label': <dict>, # <label:str>: <bbox_template>
+        'bbox_distribution_in_attribute': <dict>,
+        #  <label:str>: {
+        #     <attribute:str>: { <attr_value>: <bbox_template>, }
+        #  }
+        'bbox_distribution_in_dataset_item': <dict>,
+        # '(<item.id>, <item.subset>)': <bbox count: int>
+
+        ## statistics for segmentation task
+        'items_with_invalid_value'] = <dict>,
+        #  '(<item.id>, <item.subset>)' : {
+        #    <ann.id> : [ <property:str>, ]
+        #  }
+        #  properties: 'area', 'width', 'height'
+        'mask_distribution_in_label'] = <dict>, # <label:str>: <mask_template>
+        'mask_distribution_in_attribute'] = <dict>,
+        #  <label:str>: {
+        #     <attribute:str>: { <attr_value>: <mask_template>, }
+        #  }
+        'mask_distribution_in_dataset_item'] = <dict>,
+        # '(<item.id>, <item.subset>)': <mask/polygon count: int>
+    },
+    'validation_reports': <list>, #[ <validation_error_format>, ]
+    # validation_error_format = {
+    #   'anomaly_type': <str>,  # see datumaro/components/errors.py
+    #   'description': <str>,   # see datumaro/components/errors.py
```

Comment:
Who is supposed to read the code?

Reply:
Do you mean the anomaly_type should be documented, instead of referring to the code?

Reply:
I thought that this information is needed to parse the validation result.